test/runtests.jl

```diff
@@ -14,21 +14,34 @@ using ExponentialUtilities: getH, getV, _exp!
     @test A2 ≈ expA2
 end
 
-@testset "exp_generic" begin 
+@testset "exp_generic" begin
     for n in [5, 10, 30, 50, 100, 500]
         M = rand(n, n)
         @test exp(M) ≈ exp_generic(M)
 
         M′ = M / 10opnorm(M, 1)
         @test exp(M′) ≈ exp_generic(M′)
-        
+
         N = randn(n, n)
         @test exp(N) ≈ exp_generic(N)
 
         exp(n) ≈ exp_generic(n)
     end
 end
 
+# The issue was only triggered if exp_generic had been called before loading
```

Comment:
This should be incorrect. What matters is the load order, while it's a bug if the call order is observable.

Reply:
how'd you find this? 😆 I think we can just straight avoid the issue for the most common case (https://github.com/SciML/ExponentialUtilities.jl/pull/45) and handle the more general thing later.

Reply:
Someone put out the bat signal 🙃